UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -45,165 +45,127 @@
 using Newtonsoft.Json.Linq;
 using System.Net;
 using System.Globalization;
+using System.Windows.Controls.Primitives;
 
 namespace UndertaleModTool
 {
+    [DebuggerDisplay(@"\{{GetType().FullName,nq} - {OpenedObject}\}")] // display value in debugger as "{UndertaleModTool.Tab - {*object*}}"
```

Comment:
Why not just use a custom ToString rather than using a specific debugger display?

Reply:
I didn't know that Visual Studio using `ToString()` for value displaying, so yeah, it will be better.